helm/app/values.yaml

```diff
@@ -27,7 +27,7 @@ ingress:
     nginx.ingress.kubernetes.io/configuration-snippet: |
       rewrite ^(/dashboard)$ $1/ permanent;
   hosts:
-    - host: go-api-boilerplate.local
+    - host: go-api-boilerplate.me
```

Comment:
why .me ?
.local is considered best practice for localhost alias, even when u read k8s docs they use it
https://kubernetes.io/docs/concepts/services-networking/add-entries-to-pod-etc-hosts-with-host-aliases/

Reply:
In the case of google .local domain is not accepted as a valid Authorized redirect URIs public domain. I suppose it will be the same for all other social providers.

> Invalid Redirect: must end with a public top-level domain (such as .com or .org).